src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php

```diff
@@ -355,6 +355,9 @@ public function openAction(Request $request)
         $file = $request->query->get('file');
         $line = $request->query->get('line');
 
+        $flags = PHP_VERSION_ID >= 50400 ? ENT_QUOTES | ENT_SUBSTITUTE : ENT_QUOTES;
+        $file =  nl2br(htmlspecialchars($request->query->get('file'), $flags, 'UTF-8')); 
```

Comment:
why `nl2br` here ? This is not required for escaping.

And I would reuse `$file`

Reply:
hello, thanks for your comment, I use that function by default to avoid XSS. If you think it's unnecessary we can consider removing it. Thank you

Reply:
> And I would reuse `$file`

You are right, it would be more optimal to apply the modification within the IF block